packages/api/src/server.js

```diff
@@ -70,14 +72,21 @@ const documentRoute = require('./routes/documentRoute');
 const taskRoute = require('./routes/taskRoute');
 const productRoute = require('./routes/productRoute');
 const notificationUserRoute = require('./routes/notificationUserRoute');
+const { use } = require('chai');
 
 // register API
 const router = promiseRouter();
 
-app.get('/', (req, res) => {
+app.get('/', async (req, res) => {
```

Comment:
Why add `async`?

Reply:
I just checked the branch, for some reason ./cypress.env.json i'd added to the .gitignore file still appears

Reply:
I think i'd hard coded the clean db tables route to that route when i was testing and forgot to return it to its original state

Reply:
Adding a file to `.gitignore` won't remove it from the repo if it was committed before.

Reply:
Understood, my assumption was that since this is a new branch those files would be ignored when i pushed my first commit